Sources/Utility/ArgumentParser.swift

```diff
@@ -193,15 +214,15 @@ public final class PositionalArgument<Kind>: ArgumentProtocol {
 /// A type-erased argument.
 ///
 /// Note: Only used for argument parsing purpose.
-fileprivate final class AnyArgument: ArgumentProtocol, CustomStringConvertible {
+public final class AnyArgument: ArgumentProtocol, CustomStringConvertible {
```

Comment:
In order to generate those scripts, meta information is required about the possible arguments, usage, overview and sub parsers. What do you propose where this information should come from?